### DISCUSSION TITLE
Nuxt 2. ethereum-provider. Missing loader.

### DISCUSSION THREAD

**Discussion by alxbychkov** [WalletConnect/walletconnect-monorepo#3580] [Developer Support] [answered]
Hi, i use nuxt 2 app. After install package i see an issue about loader for this type files.
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/51967476/b5186b30-24e6-4684-851c-d2e77804b08b)

What loader need to install? 

Thanks

**Comment by glitch-txs** [comment, marked as answer]
hey, could you try using CDN? or upgrading your bundler configuration:
https://github.com/glitch-txs/ethereum-provider-cdn

**Comment by KBKUN024** [comment]
@alxbychkov Have you just add a cdn to resolve this issue?

**Reply by alxbychkov** [reply]
Yes, i'am using this ones:
`import {
  EthereumClient,
  w3mConnectors,
  w3mProvider,
  WagmiCore,
  WagmiCoreChains,
  WagmiCoreConnectors,
} from 'https://unpkg.com/@web3modal/ethereum@2.7.1';

import { Web3Modal } from 'https://unpkg.com/@web3modal/html@2.7.1';`

**Reply by KBKUN024** [reply]
Oh,thanks for you reply.Where did you add these codes? Did you create a file in the plugins folder and then configure plugins in nuxt.config.js? Or somewhere else? Can I see your code?

**Reply by alxbychkov** [reply]
i create static js file, with this logic. and include it component (head)

**Reply by KBKUN024** [reply]
But I can't import file from CDN,I tried your code, console will shows like: 
This dependency was not found:                                                                                                                                                                             
* https://unpkg.com/@web3modal/ethereum@2.7.1 in ./plugins/walletconnect-ethereum-provider.js   
Have you met this error like me?

**Reply by alxbychkov** [reply]
Try to use static folder

**Reply by KBKUN024** [reply]
Ok,I try it now.

**Reply by KBKUN024** [reply]
In static folder,I created a file,named : ethereumprovider.js
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/54589019/6a6893d3-9618-4fa1-8228-cc42cc178bab)
And then I import pkg in another page, console shows a error:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/54589019/80895c2b-2f2e-4c22-94f6-574290664a0f)
Do you know how to fix it?

**Reply by alxbychkov** [reply]
It works in js file, without import in another component? i think will not work like this.

**Reply by KBKUN024** [reply]
If I just import like this:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/54589019/10fb9c76-160a-41ca-94b8-3da4ccf77779)
Then how should I use it? Or can you show me your code?I really want to fix this issue,it confuses me for a long time.

**Reply by alxbychkov** [reply]
Sorry, my project have NDA, i could not share it.

I do logic in this file. Write functions to window and use them in components.

**Reply by KBKUN024** [reply]
Write functions to window and use them in components. How to do that, you can just share part of it or tell me how to do it(you can use virtual code,not your code of your project)

**Reply by alxbychkov** [reply]
What's your project? What do you want to solve?

**Reply by KBKUN024** [reply]
I'm also a nuxt2 project,and I'm also having the same issue as the title of your issue,so I want to solve it , I installed @walletconnect/ethereum-provider,but I can't import it in a vue file,console told me:the dependency not found

**Reply by alxbychkov** [reply]
create test repo with nuxt 2 project and share it, i will try later

**Reply by KBKUN024** [reply]
ok,thanks a lot!I'll create a repo soon.

**Reply by KBKUN024** [reply]
this is my test repo: [https://github.com/KBKUN024/nuxt2-test-project](url) , if you have time,please help me how to fix it,thanks a lot again!
